### DISCUSSION TITLE
formState.submitCount

### DISCUSSION THREAD

**Discussion by fachrur95** [react-hook-form/react-hook-form#6925] [Q&A] [answered]
I have problem on formState.submitCount. counter is still count if form validate error. how can i solve it? thanks before.

**Comment by ryanmk54** [comment, marked as answer]
According to this discussion, https://github.com/react-hook-form/react-hook-form/discussions/2793, counter is incremented even if submission failed.

**Reply by bluebill1049** [reply]
Thanks @ryanmk54 This is a grey area, there is no standard definition of submitting count, a failed submission is a submit as well. so unless we can have a standard definition, we will either keep it as it is or consider fixing it as a bug if this is incorrect.

**Reply by fachrur95** [reply]
Ohh I got the point. okay, I think this option can help me about prevent many submit. So, if this option is grey area, I will prevent manually. Thanks a lot about this answer

**Comment by ryanmk54** [comment]
form.formState has a boolean property isSubmitSuccessful
